sandbox/apparmor/apparmor.go

```diff
@@ -486,7 +486,7 @@ func AppArmorParser() (cmd *exec.Cmd, internal bool, err error) {
 	for _, dir := range filepath.SplitList(parserSearchPath) {
 		path := filepath.Join(dir, "apparmor_parser")
 		if _, err := os.Stat(path); err == nil {
-			return exec.Command(path), false, nil
+			return exec.Command(path, "--policy-features=<kernel>"), false, nil
```

Comment:
this needs a long comment explaining `<..>` meaning.

Reply:
https://gitlab.com/apparmor/apparmor/-/blob/master/parser/parser_main.c#L638

Reply:
I agree that we need a better grasp of what the options mean. There are more things we could pass and I'm not sure which ones we really should and should not touch.

Reply:
https://gitlab.com/apparmor/apparmor/-/commit/162da1ba48635a3c0be8390bd405105d700c715f gives some insight